src/codeu/chat/client/commandline/Chat.java

```diff
@@ -24,6 +24,8 @@
 import codeu.chat.client.core.MessageContext;
 import codeu.chat.client.core.UserContext;
 
```

Comment:
nit: remove newline, all imports should be one block
https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing